tests/components/cover/test_device_trigger.py

```diff
@@ -585,118 +602,110 @@ def calls(hass):
-                            )
+                            "some":
+                            ("is_pos_lt_90 - {{ trigger.platform}} - "
+                             "{{ trigger.entity_id}} - {{ trigger.from_state.state}} - "
+                             "{{ trigger.to_state.state}} - {{ trigger.for }}")
                         },
                     },
                 },
                 {
-                    "trigger": [
-                        {
-                            "platform": "device",
-                            "domain": DOMAIN,
-                            "device_id": "",
-                            "entity_id": ent.entity_id,
-                            "type": "tilt_position",
-                            "above": 45,
-                            "below": 90,
-                        }
-                    ],
+                    "trigger": [{
+                        "platform": "device",
+                        "domain": DOMAIN,
+                        "device_id": "",
+                        "entity_id": ent.entity_id,
+                        "type": "tilt_position",
+                        "above": 45,
+                        "below": 90,
+                    }],
                     "action": {
                         "service": "test.automation",
                         "data_template": {
-                            "some": (
-                                "is_pos_gt_45_lt_90 - {{ trigger.platform}} - "
-                                "{{ trigger.entity_id}} - {{ trigger.from_state.state}} - "
-                                "{{ trigger.to_state.state}} - {{ trigger.for }}"
-                            )
+                            "some":
+                            ("is_pos_gt_45_lt_90 - {{ trigger.platform}} - "
+                             "{{ trigger.entity_id}} - {{ trigger.from_state.state}} - "
+                             "{{ trigger.to_state.state}} - {{ trigger.for }}")
                         },
                     },
                 },
             ]
         },
     )
-    hass.states.async_set(
-        ent.entity_id, STATE_CLOSED, attributes={"current_tilt_position": 50}
-    )
+    hass.states.async_set(ent.entity_id,
+                          STATE_CLOSED,
+                          attributes={"current_tilt_position": 50})
     await hass.async_block_till_done()
     assert len(calls) == 3
-    assert sorted(
-        [calls[0].data["some"], calls[1].data["some"], calls[2].data["some"]]
-    ) == sorted(
-        [
-            "is_pos_gt_45_lt_90 - device - cover.set_position_cover - open - closed - None",
-            "is_pos_lt_90 - device - cover.set_position_cover - open - closed - None",
-            "is_pos_gt_45 - device - cover.set_position_cover - open - closed - None",
-        ]
-    )
-
-    hass.states.async_set(
-        ent.entity_id, STATE_CLOSED, attributes={"current_tilt_position": 95}
-    )
+    assert sorted([
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)